quinn-proto/src/connection/mod.rs

```diff
@@ -393,11 +393,6 @@ where
             }
         }
 
-        if self.path.anti_amplification_blocked(self.path.mtu.into()) {
```

Comment:
What is the reason for removing it here? I mean it definitely is also checked later on and therefore not necessary. But apart from that, will the short circuit prevent here anything?

Reply:
Just trying to simplify one of our most complex functions. There is a slight change in behavior in that `ensure_probe_queued` might run earlier, but that should be harmless, and could be addressed by moving it into the loop below in any case if we decide to care.